com.neoutils.NeoRegex.yml

```diff
@@ -0,0 +1,33 @@
+app-id: com.neoutils.NeoRegex
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: NeoRegex
+
+finish-args:
+  - --socket=x11
+  - --device=dri
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+
+  - name: NeoRegex
+    buildsystem: simple
+    build-commands:
+      - ls files
+      - install -Dm755 files/bin/NeoRegex /app/bin/NeoRegex
+      - mkdir -p /app/lib
+      - cp -r files/lib/* /app/lib/
+      - install -Dm644 files/lib/NeoRegex.png /app/share/icons/hicolor/192x192/apps/com.neoutils.NeoRegex.png
+      - install -Dm644 files/com.neoutils.NeoRegex.desktop /app/share/applications/com.neoutils.NeoRegex.desktop
+      - install -Dm644 files/com.neoutils.NeoRegex.metainfo.xml /app/share/metainfo/com.neoutils.NeoRegex.metainfo.xml
+    sources:
+      - type: archive
+        url: https://github.com/NeoUtils/NeoRegex/releases/download/v2.0.0/NeoRegex-2.0.0.tar.gz
```

Comment:
This is x86_64 only:

```
file libapplauncher.so 
libapplauncher.so: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=7b0fbf9e992648c4c9b7d4dea692450d66b14b04, with debug_info, not stripped
```

Reply:
https://docs.flathub.org/docs/for-app-authors/requirements#flathubjson

Reply:
Done.

Reply:
why not building from source instead and have aarch64 support?